### DISCUSSION TITLE
Problems with .transcribe function

### DISCUSSION THREAD

**Discussion by mryt66** [openai/whisper#1923] [Q&A]
I get FileNotFoundError while the file is definitly in this folder but it still can't find it somehow, can't figure out why.
![image](https://github.com/openai/whisper/assets/64143856/c68c99d6-f88a-431e-bf7b-451ae77d80b5)

**Comment by glangford** [comment]
You are posting to the wrong project. `whisper-timestamped` has a different API, you should ask over there.
